### DISCUSSION TITLE
I use lavalink=4.0.0 and wavelink.py =3.1.0 .  I can't use wavelink to connect the lava server.

### DISCUSSION THREAD

**Discussion by TwoKrugs** [lavalink-devs/Lavalink#995] [Q&A] [answered]
import discord
from discord.ext import commands
import wavelink
intents = discord.Intents.all()
bot = commands.Bot(command_prefix='!',intents=intents)

@bot.event
async def on_ready():
    print(f'Logged in as {bot.user.name}')
    print('------')

    await wavelink.NodePool.create_node(
        host='http://127.0.0.1/',
        port=80,
        password='123456',
        bot=bot
    )
bot.run("Token")
---------------------------------------------------------------
I also use the application.yml file.
My code always had this problem:
### Connection Failure: Cannot connect to host http:80 ssl:default [getaddrinfo failed]
Hoping someone can help me

**Comment by topi314** [comment, marked as answer]
usually lavalink runs on port `2333` by default
we can't really help you without logs or the `application.yml` file

make sure your lavalink is accessible from where you run your bot
